graylog2-web-interface/src/components/search/SidebarMessageField.jsx

```diff
@@ -18,6 +17,27 @@ const SidebarMessageField = React.createClass({
   _toggleShowActions() {
     this.setState({showActions: !this.state.showActions});
   },
+  _setShowActions(isOpen) {
+    this.setState({showActions: isOpen});
+  },
```

Comment:
Because `_setShowOptions` receives the `isOpen` value and this is needed to have sane behaviour of the arrow icon when closing the dropdown by clicking somewhere else or pressing escape. :wink: